src/libraries/Microsoft.Extensions.Logging/src/LoggerFactoryOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// The options for a LoggerFactory.
+    /// </summary>
+    public class LoggerFactoryOptions
+    {
+        /// <summary>
+        /// Creates a new <see cref="LoggerFactoryOptions"/> instance.
+        /// </summary>
+        public LoggerFactoryOptions() { }
```

Comment:
Do you need this?

Reply:
Yes, this is the [design guidelines](https://docs.microsoft.com/sl-si/dotnet/standard/design-guidelines/constructor) recommendation:

✔️ DO explicitly declare the public parameterless constructor in classes, if such a constructor is required.

Reply:
I need a shortcut for pasting gifs into github